tools/docker/quorum-multi-party-all-in-one/healthcheck.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+# Besu Member 1
+wget -O- --post-data '{"jsonrpc":"2.0","method":"eth_chainId","params":[],"id":1}' http://localhost:20000
```

Comment:
@travis-payne Let me know once the health check has been updated and then it's LGTM